libraries/APM_Control/AP_YawController.cpp

```diff
@@ -79,11 +79,7 @@ int32_t AP_YawController::get_servo_out(float scaler, bool disable_integrator)
 	}
 	_last_t = tnow;
 	
-
-    int16_t aspd_min = aparm.airspeed_min;
-    if (aspd_min < 1) {
-        aspd_min = 1;
-    }
+    float aspd_min = MAX(airspeed.get_airspeed_min(),1.0f);
```

Comment:
What's wrong with this?
On May 28, 2016 11:12, "Tom Pittenger" <notifications@github.com> wrote:

> In libraries/APM_Control/AP_YawController.cpp
> <https://github.com/ArduPilot/ardupilot/pull/4229#discussion_r64992156>:
>
> > @@ -79,11 +79,7 @@ int32_t AP_YawController::get_servo_out(float scaler, bool disable_integrator)
> >  	}
> >  	_last_t = tnow;
> >  	
> > -
> > -    int16_t aspd_min = aparm.airspeed_min;
> > -    if (aspd_min < 1) {
> > -        aspd_min = 1;
> > -    }
> > +    float aspd_min = MAX(airspeed.get_airspeed_min(),1.0f);
>
> Min
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ArduPilot/ardupilot/pull/4229/files/0c32718a179cef617dd50e00bd13465e89bda228#r64992156>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAipiCaqRJgzoLiKUBhdeY_ZbTn9Muzkks5qGIWZgaJpZM4IpGHh>
> .
>